components/RichBody/index.tsx

```diff
@@ -20,36 +20,60 @@ interface Props {
   classes?: string; // custom styles for regular text (color, font weight, etc.)
 }
 
+interface DirectionObject {
+  props: { dir: string; children: any };
+}
+
 export default function RichBody(props: Props): ReactElement {
   const { body, headingClasses, classes } = props;
+
+  const onDirection = (obj: DirectionObject) => {
```

Comment:
This function name could use more context I suggest `getTextDirection`. Also we can move it outside of the component since it doesn't rely on any of it's properties. This way it isn't created every time we render 😄